packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -64,16 +110,15 @@ class AccountImportOptions extends Component {
 
   handleChangeFile = async jsonString => {
     const {
-      createAccountStore,
-      createAccountStore: { setJsonString }
+      createAccountStore: { address, setJsonString }
     } = this.props;
 
     this.setState({ isLoading: true });
 
     try {
       await setJsonString(jsonString);
 
-      if (this.hasExistingAddressForImport(createAccountStore.address)) {
+      if (this.hasExistingAddressForImport(address)) {
```

Comment:
this change needs to be reverted. it makes handleChangeFile throw an error and makes it impossible to recover from a JSON keyfile.

When trying to recover from JSON keyfile:
```
index.js:1446 TypeError: Cannot read property 'toLowerCase' of null
    at AccountImportOptions.js:166
```
`address` is null because it was retrieved before setting the json string.